### DISCUSSION TITLE
minecraft.jar location?

### DISCUSSION THREAD

**Discussion by SirenGlitch** [PrismLauncher/PrismLauncher#900] [Q&A] [answered]
Anyone know where prism stores minecraft.jar files? Seeing as you can as much as replace them from the app it should be easier to find them

**Comment by TheKodeToad** [comment, marked as answer]
The jars can be found in `libraries/com/mojang/minecraft` within the data directory.

**Reply by SirenGlitch** [reply]
Thanks! Probably wouldn't have found this alone

**Reply by TheKodeToad** [reply]
You're welcome!

**Comment by bellakeys** [comment]
Where would one find this one macos?

**Reply by SirenGlitch** [reply]
Probably the same place, `%prism install folder%/libraries/com/mojang/minecraft`

**Reply by bellakeys** [reply]
Thank you! it was a bit of digging but i found it !
